README.md

```diff
@@ -18,4 +27,9 @@ Then you can use the plugin just like one of TinyMCE's builtin plugins. When con
  `extended_valid_elements` to allow `span` tags with the `lang` attribute or the `id` attribute and to strip empty `span
  ` tags:
 
-`extended_valid_elements: 'span[lang|id] -span'`
\ No newline at end of file
+`extended_valid_elements: 'span[lang|id] -span'`
+
+# Caveats
+- The plugin doesn’t completely work: you can’t select and change the language of existing text in a list.
```

Comment:
I feel like this can be worded a bit nicer. `There is a known limitation of the plugin...` Something like that?